pkgs/applications/editors/jetbrains/plugins/default.nix

```
@@ -106,17 +107,18 @@ rec {
               sed "s|${ide}|$out|" -i $out/bin/.goland-wrapped
             '';
           };
+          patchElfCmd = if !stdenv.isDarwin then "autoPatchelf \"$out/${meta.mainProgram}/bin\"" else "";
```

Similiar to the others

```suggestion
          patchElfCmd = lib.optionalString stdenv.isLinux "autoPatchelf \"$out/${meta.mainProgram}/bin\"";
```